### PR TITLE
add model as a service workflows for the client side

### PR DESCRIPTION
**Pull request by ImenBizid**
Model as a service bucket:
1. Proactive
Start Model Service
Deploy Model
Call Prediction Service
Model Service Actions
2. Proactive Examples
Deploy and Call IRIS Flowers Classifier Model

Model as Service PCA side:
Model_Service
Resume_Model_Service
Finish_Model_Service
Pause_Model_Service

### CODE REVIEW

File: ModelAsServiceResources/METADATA.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+					"contentType": "text/x-python"
+				},
+				"file" : "resources/catalog/ml_service.py"
+			},
+			{
+				"name" : "ml_service-api",
+				"metadata" : {
+					"kind": "text",
+					"commitMessage": "First commit",
+					"contentType": "application/octet-stream"
```
Comment (andrewssobral) [inline ModelAsServiceResources/METADATA.json]: @ImenBizid why `application/octet-stream` instead of `text/x-yaml` ?

**Comment by fviale** [comment]
Could you please add the references to one of the new fork environment script (in groovy) ?

File: ModelAsServicePCA/resources/catalog/Model_Service.xml [hunk 2]
```diff
@@ -209,9 +212,9 @@ It will run every minute. ]]>
         <task ref="Start_Model_Service"/>
       </depends>
       <inputFiles>
-        <files accessMode="transferFromGlobalSpace" includes="cloud-automation-service-client-8.2.0-SNAPSHOT.jar"/>
+        <files  includes="cloud-automation-service-client-8.2.0-SNAPSHOT.jar" accessMode="transferFromGlobalSpace"/>
```
Comment (fviale) [inline ModelAsServicePCA/resources/catalog/Model_Service.xml]: are you sure of this? It seems very dangerous to use the cloud-automation-service-client of an old version...

File: ModelAsServiceResources/METADATA.json (hunk 1 above)
Comment (ImenBizid) [reply ModelAsServiceResources/METADATA.json]: For now, tests are working fine even with `application/octet-stream` but I will add the `text/x-yaml` content type in the catalog later.

File: ModelAsServicePCA/resources/catalog/Model_Service.xml (hunk 2 above)
Comment (ImenBizid) [reply ModelAsServicePCA/resources/catalog/Model_Service.xml]: You are right. Indeed, no need to add the jar in the data management of the PCA service anymore. I will delete this.

**Pull request merged**